{"uuid": "7e7a162f-2afa-4c2a-9389-1b0386e577db", "children": ["d43668ee-78f6-4dd1-919c-fd82bcfbfd12", "882461b2-4aa8-4398-a7ea-9c4c7eaba017", "18e99c86-11ed-47cf-8f3b-aafe4fba2bef", "5e697fc3-cf6f-446e-ba6f-d935377abab8", "620d82ed-7fbf-4b80-8849-2051afecd9c6", "fe52fe7c-534f-4202-9b0e-1530a3d45903", "d991baa1-d251-42fd-b795-50929b5b1e93", "6f3f484e-bdf2-4e3f-ad70-f15ca06a345c", "33aedcbe-d46e-48a4-8771-01f7b65f8c91"], "befores": [{"name": "get_job_details", "status": "broken", "statusDetails": {"message": "requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pluggy-1.5.0-py3.10.egg/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pytest-7.4.3-py3.10.egg/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/pytest-7.4.3-py3.10.egg/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/tests/Test_API_Suite/test_catena.py\", line 16, in get_job_details\n    peviitor_jobs_data = testutils.scrape_peviitor(company_name, 'Rom√¢nia')\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/tests/utils.py\", line 70, in scrape_peviitor\n    response_data = TestUtils._get_request(params)\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/tests/utils.py\", line 48, in _get_request\n    updateapi.get_token()\n  File \"/home/runner/work/scrapers_Larisa_M16/scrapers_Larisa_M16/sites/__utils/peviitor_update.py\", line 42, in get_token\n    self.access_token = requests.request(\"POST\", \"https://api.peviitor.ro/v5/get_token/\", headers=post_header, data=payload).json()['access']\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/requests-2.28.2-py3.10.egg/requests/models.py\", line 975, in json\n    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)\n"}, "start": 1714108492918, "stop": 1714108495128}], "start": 1714108492918, "stop": 1714108495566}